src/assets/custom/css/_sass/_services-page-intro.scss

```diff
@@ -1,23 +1,143 @@
+$header-background-nav-height--large-and-extra-large: $vertical-overlap * 4;
+$header-background-nav-image-height--large-and-extra-large: $vertical-overlap * 5;
+$header-background-nav-image-height--medium: 300px;
```

Comment:
This guy seems to have been left out from your `$vertical-overlap` rythm you've set up.